Dockerfile

```diff
@@ -6,7 +7,7 @@ COPY ./requirements.txt /app
 COPY ./run.sh /app
 
 RUN apt-get update &&\
-    apt-get install -y curl jq
+  apt-get install -y curl jq
```

Comment:
This needs to be tested ASAP.

The Variables that are brought in from terraform need to be manually added. This is a slightly worse experience than the script.

Reply:
Sorry that is if deploying to the cloud